source/ht_file.cpp

```diff
@@ -78,8 +78,7 @@ namespace Hatchit
         File::File(void)
             : IFile(),
             m_position(0),
-            m_size(0),
-            m_handle(nullptr) {}
+            m_size(0) {}
```

Comment:
No, the default constructor isn't called.  RAII works automatically with destructors, but you still need to be explicit with constructors.  i.e. If you don't call the constructor on a pointer, then the pointer isn't guaranteed to be nullptr, or guaranteed any value.